src/runtime/node/url/index.ts

```diff
@@ -33,29 +54,31 @@ export const resolve = <typeof url.resolve>function (from, to) {
 };
 
 // https://nodejs.org/api/url.html#url_url_urltohttpoptions_url
-export const urlToHttpOptions = <typeof url.urlToHttpOptions>function (url) {
-  return {
-    protocol: url.protocol,
-    hostname: url.hostname,
-    hash: url.hash,
-    search: url.search,
-    pathname: url.pathname,
-    path: url.pathname + url.search || "",
-    href: url.href,
-    port: url.port,
+export const urlToHttpOptions = <typeof nodeUrl.urlToHttpOptions>(
+  function (url) {
+    return {
+      protocol: url.protocol,
+      hostname: url.hostname,
+      hash: url.hash,
+      search: url.search,
+      pathname: url.pathname,
+      path: url.pathname + url.search || "",
+      href: url.href,
+      port: url.port,
 
-    auth: url.username
-      ? url.username + url.password
-        ? ":" + url.password
-        : ""
-      : "",
-  };
-};
+      auth: url.username
+        ? url.username + url.password
+          ? ":" + url.password
+          : ""
+        : "",
+    };
+  }
+);
 
 // https://nodejs.org/api/url.html#url_url_format_urlobject
 // TODO
-export const format = <typeof url.format>(
-  function (urlInput, options?: url.URLFormatOptions) {
+export const format = <typeof nodeUrl.format>(
```

Comment:
could you write this as, which actually typechecks the right-handside as opposed to the original which just casts the value without typechecking?
```suggestion
export const format: typeof nodeUrl.format = (
```

Reply:
they are same (i did it for consistency with rest of file ... which is not best format!)